config/environment_variables_template.tfbackend

```diff
@@ -0,0 +1,7 @@
+
+# Create <name-of-environment>.tfbackend for every environment that You intend
+# to deploy from this template:
+
+"bucket" =
+"key"    = "terraform/Infrastructure/<name-of-environment>.tfstate"
```

Comment:
Improvement: We could use terraform workspaces instead of hardcoding this
https://www.terraform.io/docs/state/workspaces.html